### DISCUSSION TITLE
wslupath should not be deprecated

### DISCUSSION THREAD

**Discussion by eggbean** [wslutilities/wslu#296] [General]
I don't think `wslupath` should be deprecated until `wslpath` sorts out this deficiency where it cannot make Windows paths with double backslashes like this tool can (`--doubledash-dir`)!

**Comment by car-meows** [comment]
Unfortunately `wslu` is already in maintenance mode; Hence I am not looking to continue working on `wslupath`. it is where mostly for backward compatibilities.
